### PR TITLE
Manage Packet Broker gateway visibility

### PR DESCRIPTION
**Pull request by johanstokking**
#### Summary

References #4409; per-network visibility management is tracked in this issue too and we don't implement this yet. The default visibility management is part of this PR, and only CLI.

#### Changes

Manage Packet Broker default gateway visibility with the CLI.

#### Testing

Local integration testing

##### Regressions

None expected

#### Notes for Reviewers

If you have access to Packet Broker (staging or prod) you can test this already; Packet Broker has been supporting this API for a while now.

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [ ] Documentation: Relevant documentation is added or updated.
- [x] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

File: cmd/ttn-lw-cli/commands/packetbroker.go [hunk 1]
```diff
@@ -149,6 +149,43 @@ func getPacketBrokerRoutingPolicy(flagSet *pflag.FlagSet) (up ttnpb.PacketBroker
...
+		res.Location = true
+		res.AntennaPlacement = true
+		res.AntennaCount = true
+		res.ContactInfo = true
+		res.Status = true
+		res.FrequencyPlan = true
+		res.PacketRates = true
+	}
+	return res
+}
```
Comment (htdvisser) [inline cmd/ttn-lw-cli/commands/packetbroker.go]: We have utilities that can extract this from and set this to the proto message, why not use those?

File: cmd/ttn-lw-cli/commands/packetbroker.go [hunk 2]
```diff
@@ -379,6 +416,100 @@ for the Home Network (by NetID and tenant ID).`,
...
+  To get the default gateway visibility:
+    $ ttn-lw-cli packetbroker home-network gateway-visibilities get default`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			id, err := getPacketBrokerNetworkID(cmd.Flags(), args, true)
+			if err != nil {
+				return err
+			}
+			// TODO: Support per-network settings (https://github.com/TheThingsNetwork/lorawan-stack/issues/4409)
+			if id != nil {
+				return errPacketBrokerNetworkID
```
Comment (adriansmares) [inline cmd/ttn-lw-cli/commands/packetbroker.go:403]: Nit: `.New()`

File: cmd/ttn-lw-cli/commands/packetbroker.go (hunk 1 above)
Comment (adriansmares) [reply cmd/ttn-lw-cli/commands/packetbroker.go]: This is probably for consistency reasons, given that the whole file uses this style.

**Review by adriansmares — APPROVED** [review]

File: cmd/ttn-lw-cli/commands/packetbroker.go (hunk 2 above)
Comment (johanstokking) [reply cmd/ttn-lw-cli/commands/packetbroker.go:403]: We don't use `.New()` in CLI

**Pull request merged**